packages/bitcore-wallet-client/src/lib/request.ts

```diff
@@ -161,7 +185,7 @@ export class Request {
         ret = new Error(
           body.code +
             ': ' +
-            (_.isObject(body.message)
+            (typeof body.message === 'object'
```

Comment:
typeof null === 'object' returns true
Maybe we can replace it with: typeof body.message === 'object' && body.message !== null

Reply:
And then: body.message ?? ''

Reply:
Good call - missed that one

Reply:
> And then: body.message ?? ''

Do you mean add a coalesce after the tertiary statement like 
```
(body.message && typeof body.message === 'object'
              ? JSON.stringify(body.message)
              : body.message ?? 'Unknown error')
```
?

Reply:
yes, that's fine